src/offset/local/windows_sys/mod.rs

```diff
@@ -0,0 +1,151 @@
+use std::{fmt, io::Error, ptr, result::Result};
+
+use winapi::shared::minwindef::*;
+use winapi::um::minwinbase::SYSTEMTIME;
+use winapi::um::timezoneapi::*;
```

Comment:
Would like to get rid of the `*` imports here, can you expand them to what we're actually using?